src/core/openthread-core-default-config.h

```diff
@@ -912,7 +912,54 @@
 #define OPENTHREAD_CONFIG_ENABLE_STEERING_DATA_SET_OOB          0
 #endif
 
-/*
+/**
+ * @def OPENTHREAD_CONFIG_CHANNEL_MONITOR_SAMPLE_INTERVAL
+ *
+ * The sample interval in milliseconds used by Channel Monitoring feature.
+
+ * When enabled every sample interval a zero-duration Energy Scan is performed on every channel collecting a single
+ * RSSI sample per channel.
+ *
+ * Applicable only if Channel Monitoring feature is enabled (i.e., `OPENTHREAD_ENABLE_CHANNEL_MONITOR` is set).
+ *
+ */
+#ifndef OPENTHREAD_CONFIG_CHANNEL_MONITOR_SAMPLE_INTERVAL
+#define OPENTHREAD_CONFIG_CHANNEL_MONITOR_SAMPLE_INTERVAL       41000
+#endif
+
+/**
+ * @def OPENTHREAD_CONFIG_CHANNEL_MONITOR_RSS_THRESHOLD
+ *
+ * The RSSI threshold in dBm used by Channel Monitoring feature.
+ *
+ * The RSSI samples are compared with the given threshold. Channel monitoring reports the average rate of RSS samples
+ * that are above this threshold within a certain observation window (per channel).
+ *
+ * It is recommended that this value is set to same value as the CCA threshold used by radio.
+ *
+ * Applicable only if Channel Monitoring feature is enabled (i.e., `OPENTHREAD_ENABLE_CHANNEL_MONITOR` is set).
+ *
+ */
+#ifndef OPENTHREAD_CONFIG_CHANNEL_MONITOR_RSS_THRESHOLD
+#define OPENTHREAD_CONFIG_CHANNEL_MONITOR_RSS_THRESHOLD        -75
+#endif
+
+/**
+ * @def OPENTHREAD_CONFIG_CHANNEL_MONITOR_RSS_THRESHOLD
+ *
+ * The averaging sample window length (multiple of sample interval) used by Channel Monitoring feature.
```

Comment:
nit: I originally read "multiple" as a value equal to `X * OPENTHREAD_CONFIG_CHANNEL_MONITOR_SAMPLE_INTERVAL`, when in fact this is just `X`.  How about saying that this is the window size in units of channel sample intervals?